### PR TITLE
task 5

### PR DESCRIPTION
**Pull request by GrowYourOwnFreedom**
task 5 get reviews complete!

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -39,21 +39,19 @@ describe("GET /api/reviews/:review_id", () => {
...
-							"https://images.pexels.com/photos/258045/pexels-photo-258045.jpeg?w=700&h=700",
-						review_body:
-							"Cold rain pours on the faces of your team of cyclists, you pulled to the front of the pack early and now your taking on exhaustion cards like there is not tomorrow, you think there are about 2 hands left until you cross the finish line, will you draw enough from your deck to cross before the other team shoot passed? Flamee Rouge is a Racing deck management game where you carefully manage your deck in order to cross the line before your opponents, cyclist can fall slyly behind front runners in their slipstreams to save precious energy for the prefect moment to burst into the lead ",
-						category: "social deduction",
-						created_at: "2021-01-18T10:01:41.251Z",
-						votes: 10,
-					},
-				];
+				const output = {
+					review_id: 10,
```
Comment (HannahBurks) [inline __tests__/app.test.js:43]: Bear in mind, should you make any changes to the data for this article, the values may change. The only one that will remain consistant always is the review_id. I would therefore suggest using a flexible matcher here such as expect.objectContaining or .toMatchObject. and only checking the article Id is 10.

**Review by HannahBurks — APPROVED** [review]
Nice work Jack - feel free to merge, but just check my comments for ticket 4 to keep in mind :)

File: __tests__/app.test.js [hunk 2]
```diff
@@ -74,3 +72,30 @@ describe("GET /api/reviews/:review_id", () => {
...
+					expect(typeof review.owner).toBe("string");
+					expect(typeof review.title).toBe("string");
+					expect(typeof review.category).toBe("string");
+					expect(typeof review.review_img_url).toBe("string");
+					expect(typeof review.created_at).toBe("string");
+					expect(typeof review.designer).toBe("string");
+					expect(typeof review.review_id).toBe("number");
+					expect(typeof review.votes).toBe("number");
+					expect(typeof review.comment_count).toBe("string");
+					expect(typeof review.review_body).toBe("undefined");
```
Comment (dappernerddesigns) [inline __tests__/app.test.js:92]: we would never test for undefined. You could change this to expecting a property to not exist.

**Review by dappernerddesigns — APPROVED** [review]
I've suggested a change in your test. Nice join and looking good. If you make the change in your test then go ahead and merge without resubmitting.

**Pull request merged**